ERCS/erc-7518.md

```diff
@@ -0,0 +1,606 @@
+
+## Backwards Compatibility
+
+1. Compatibility with  [ERC-1155](./eip-1155.md)
+
+   [ERC-7518](./eip-7518.md) is designed to be compatible with [ERC-1155](./eip-1155.md) token standards. By leveraging the features of these widely adopted token standards, [ERC-7518](./eip-7518.md) provides a robust foundation for security token functionality.
+2. Key additional features of [ERC-7518](./eip-7518.md)
+
+In addition to the compatibility with [ERC-1155](./eip-1155.md), [ERC-7518](./eip-7518.md) introduces unique features specifically tailored for security tokens. These features include:
+
+* Enhanced cross-chain interoperability: [ERC-7518](./eip-7518.md) enables security tokens to seamlessly interact with the [ERC-7518](./eip-7518.md) ecosystem, allowing for trustless transfers and integration with DeFi platforms across different blockchain networks.
+* Semi-fungible share class management: [ERC-7518](./eip-7518.md) supports the management of semi-fungible share classes, providing flexibility in the representation of ownership and fractional shares of real-world assets.
+* Token wrapping: [ERC-7518](./eip-7518.md) introduces token wrapping functionality, enabling the conversion of tokens from one standard to another, further enhancing interoperability and integration with various blockchain networks.
+* Flexible compliance management: [ERC-7518](./eip-7518.md) offers a comprehensive compliance management solution, including off-chain verification, custom rules, regulations, and signature validation. This ensures that security tokens adhere to the necessary regulatory requirements while maintaining a high level of flexibility.
+* Hard Asset such as Commercial Tower: [ERC-7518](./eip-7518.md) can be used to tokenize a commercial tower, allowing investors to own a fraction of the asset. Each `tokenId` can represent a unique share class, such as a percentage of the entire tower or individual floors. This enables greater liquidity and accessibility to real estate investments, as smaller investors can participate in owning a portion of a large commercial property
+* Soft Asset such as Funds: [ERC-7518](./eip-7518.md) can be used to tokenize funds, especially in a Fund of Funds (FoF) business model. Each sub-fund within the FoF can be tokenized, with each `tokenId` representing a unique share class. This allows investors to directly invest in specific sectors, regions, or strategies based on their preferences. The semi-fungible share class management feature of [ERC-7518](./eip-7518.md) provides the flexibility to create and manage multiple share classes with varying rights and privileges.
+
+By incorporating these additional features, [ERC-7518](./eip-7518.md) addresses the growing need for enhanced cross-chain interoperability and flexible compliance management in the rapidly evolving landscape of security tokens. This makes [ERC-7518](./eip-7518.md) a powerful and versatile token standard for the security token ecosystem.
```

Comment:
These points aren't exactly relevant to backwards compatibility. They are more motivating examples, and could be put in the motivation section.